x-pack/plugins/observability_solution/infra/public/components/asset_details/components/kpis/container_kpi_charts.tsx

```diff
@@ -38,10 +38,9 @@ export const ContainerKpiCharts = ({
   searchSessionId,
   loading = false,
 }: ContainerKpiChartsProps) => {
-  const isK8Container = useIntegrationCheck({ dependsOn: INTEGRATIONS.kubernetesContainer });
```

Comment:
Should we keep the `isK8Container` check before rendering k8s metrics in case we have a different container type (and not have k8s as default)? Maybe we can keep the integration check and show k8s data only when we have the integration enabled. I think we have k8s and docker only but I remember there could be ([containerd](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-containerd.html)) I don't think we currently support it tho so I am not sure if we can have a "default" fallback - my concern is that we assume that k8s integration is enabled t if it's not a docker conteiner this could lead to some errors in the future 🤔

Reply:
Yeah, I've been thinking about it, I think best will be checking both integrations and then return null if it's not any of them